generators/server/templates/pom.xml.ejs

```diff
@@ -628,14 +628,18 @@
             <scope>test</scope>
         </dependency>
 <%_ if (messageBroker === 'kafka') { _%>
-        <!-- Kafka support -->
         <dependency>
-            <groupId>org.springframework.kafka</groupId>
-            <artifactId>spring-kafka</artifactId>
-		</dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+        </dependency>
         <dependency>
-            <groupId>org.springframework.kafka</groupId>
-            <artifactId>spring-kafka-test</artifactId>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>database-commons</artifactId>
```

Comment:
FYI this one is not needed for Kafka I guess

Reply:
You're right! thanks.
https://github.com/jhipster/generator-jhipster/pull/10598